docs/modules/ROOT/pages/rpc.adoc

```diff
@@ -93,148 +93,148 @@ pub const ROTATE_ON_ERROR_CODES: [u16; 1] = [429];
 
 === Retry Strategy
 
-Each transport client implements `reqwest-retry` middleware with exponential backoff to handle transient failures in network requests. This is implemented separately from the endpoint rotation mechanism.
+The transport layer uses a combination of same-endpoint retries and endpoint rotation to handle transient failures and maintain service availability.
 
-* For transient HTTP errors and network failures:
-** Retries up to 3 times (configurable via `ExponentialBackoff` builder)
-** Applies exponential backoff between retry attempts
-** Returns the final error if all retry attempts fail
-** Maintains the same URL throughout the retry process
-** Independent from the endpoint rotation mechanism
+==== Same-Endpoint Retry (via `reqwest-retry`)
 
+The `HttpTransportClient` (and by extension, EVM and Stellar clients) utilizes a `reqwest_middleware::ClientWithMiddleware`. This client is configured during initialization using the `utils::http::create_retryable_http_client` utility. This utility layers `reqwest_retry::RetryTransientMiddleware` on top of a shared base `reqwest::Client`.
+
+This middleware handles:
+* Automatic retries for transient HTTP errors (e.g., 5xx server errors, network timeouts) for requests made to the *currently active RPC URL*.
+* An exponential backoff policy between these retry attempts.
+* Parameters like the number of retries, backoff durations, and jitter are defined in an `HttpRetryConfig` struct (see <<Configuration Options>>).
+* This same-endpoint retry mechanism is independent of, and operates before, the endpoint rotation logic. If all same-endpoint retries fail for the current URL, the error is then processed by the `EndpointManager`.
+
```

Comment:
I think you need an [enter] under "This middleware handles", the bullet points are not formatted properly otherwise:
![Screenshot 2025-06-13 at 11 52 22](https://github.com/user-attachments/assets/c73cf3d6-4e18-4f06-9f14-ea8320f4c880)